### ISSUE TITLE
How to counting Human 

### ISSUE THREAD

**Issue by TAThink** [EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi#74]
Hello

I built it through edgetpu.

But we need it.
You have to detect people and know how many people have passed.

Is there a way?

**Comment by ibrandiay** [comment]
yes ... you just need to loop through the set of classes found, and if there is one of the classes that matches the person class, increase your loop.

**Comment by ibrandiay** [comment]
yes ... you just have to loop through the set of classes found, and if there is one of the classes that matches the person class, increase your counter.
